pkgs/tools/games/steamtinkerlaunch/default.nix

```
@@ -14,15 +14,15 @@
 , yad
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
```

Would be better to use

```suggestion
stdenvNoCC.mkDerivation {
```